taxi/forms.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "username",
+            "first_name",
+            "last_name",
+            "license_number",
+            "email",)
```

Comment:
The `DriverCreateForm` currently lacks validation for the `license_number` field. Consider implementing a `clean_license_number` method similar to the one in `DriverLicenseUpdateForm` to ensure the license number is validated during driver creation.